src/app/core/data/relationship-data.service.ts

```diff
@@ -258,7 +260,16 @@ export class RelationshipDataService extends IdentifiableDataService<Relationshi
    * @param options
    */
   getRelatedItemsByLabel(item: Item, label: string, options?: FindListOptions): Observable<RemoteData<PaginatedList<Item>>> {
-    return this.getItemRelationshipsByLabel(item, label, options, true, true, followLink('leftItem'), followLink('rightItem'), followLink('relationshipType')).pipe(this.paginatedRelationsToItems(item.uuid));
+    let linksToFollow: FollowLinkConfig<Relationship>[];
+    if (this.appConfig.browseBy.showThumbnails) {
```

Comment:
Thanks for this @ybnd ! I sensed a code smell yesterday but had other distractions at work. I added `embedThumbnail` to `FindListOptions` (similar to how this is done for the `BrowseService)`. Now passing the parameter from` ds-related-items`.

Reply:
Perfect!

If @tdonohue agrees I sense an opportunity to refactor some of the equivalent `SearchService` etc. methods to also handle thumbnail embeds in this way (instead of depending on custom `followLink`s each time).

IMO that makes sense as a separate "nice to have" PR, as everything here is functional _without_ it and this PR is already pretty extensive.